src/tools/pub/publishing.md

```diff
@@ -41,6 +41,10 @@ are a few additional requirements for uploading a package:
   and Git dependencies don't support version resolution as well as hosted
   dependencies do.
 
+* Once you run `pub publish` you will be required to authenticate with a "Google Account". This can be a Gmail account, but you can use any Google account, including one registered with your own email address.
```

Comment:
I tried rewriting this to (1) adhere to the 80-character limit and (2) emphasize the need to have a Google Account.